openslides_backend/shared/lock.py

```diff
@@ -0,0 +1,3 @@
+import threading
```

Comment:
Please add a comment, why this code is here? What does the lock solve?

Reply:
With last commit there is more stuff in this file.

1. There is the instance of the thread.local-store to share by import: pytest_thread_local
2. There is the Context-Manager to apply and restore the MonkeyPatch for the write-method of the datastore-adapter
3. There is the write-Method to patch the original from adapter. It wraps the original write to inject the threading.lock to stop one thread and let another thread pass the first one.

The lock itself is created and controlled by the using test. An example to use this in conjunction see in tests_create_sequence_number.py Test: test_create_sequence_numbers_race_condition with race condition

Reply:
if the file is only used by tests, move it to a test-folder.

Reply:
Done